linux.md

```diff
@@ -176,7 +176,7 @@
     sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
     wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
     sudo apt update
-    sudo apt install postgresql-14
+    sudo apt install postgresql-15
```

Comment:
Hm... is `postgres-15` really an option on the Debian Apt package?

> Currently, we support
> ...
> - PostgreSQL 10, 11, 12, 13, 14, [15 beta](https://wiki.postgresql.org/wiki/Apt/FAQ#I_want_to_try_the_beta_version_of_the_next_PostgreSQL_release), [16 devel](https://wiki.postgresql.org/wiki/Apt/FAQ#Development_snapshots)

Ref: https://wiki.postgresql.org/wiki/Apt

Reply:
This blog post (from Oct. 17) seems to think that you can use it... https://www.linuxtuto.com/how-to-install-postgresql-15-on-debian-11/

Reply:
Another blog post here (also Oct 17): https://www.huuii.com/site/hacking/3655

Reply:
There are some other posts that mention this as well. For Linux support, maybe this is enough, since we'll probably have first Linux users following this guide only starting in the next months.